src/Bundle/EventListener/ORMTranslatableListener.php

```diff
@@ -26,7 +24,7 @@
 use Sylius\Component\Resource\Translation\TranslatableEntityLocaleAssignerInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
-final class ORMTranslatableListener implements EventSubscriber
+final class ORMTranslatableListener
```

Comment:
```suggestion
final class ORMTranslatableListener implements AbstractDoctrineSubscriber
```

Reply:
Do you meen extends AbstractDoctrineListener?